aries-site/src/pages/templates/popover.mdx

```diff
@@ -0,0 +1,55 @@
+import { Example } from '../../layouts';
+import { PopoverAnatomy, PopoverSimpleExample, PopoverInlineExample } from '../../examples';
+
+<Example
```

Comment:
Should we include the code under these examples?

Reply:
yes will add thank you